document feature flags available (#402)
crossplane/docs: content/master/software/install.md

-f settings.yaml
```

#### Feature flags

> Crossplane usually introduces new features behind feature flags. Alpha features

Reviewer: ```suggestion
Alpha and beta features are behind feature flags. Alpha features
```

are disabled by default. Beta features are enabled by default. To enable or

> disable a feature flag, set the `args` value in the Helm chart. Available

Reviewer: ```suggestion
turn off a feature flag, set the `args` value in the Helm chart. Available
```

> Crossplane usually introduces new features behind feature flags. Alpha features
> are disabled by default. Beta features are enabled by default. To enable or
> disable a feature flag, set the `args` value in the Helm chart. Available
> feature flags can be directly found by running `crossplane core start --help`,
> or by looking at the table below.

Author: ```suggestion
Crossplane usually introduces new features behind feature flags. By default
alpha features are off, while beta features are enabled. To enable a feature
flag, set the `args` value in the Helm chart. Available feature flags can be
directly found by running `crossplane core start --help`, or by looking at the
table below.
```

Author: pushed this, let me know if there is any other change needed @plumbis

{{< expand "Feature flags" "v" >}}
{{< table caption="Feature flags" >}}
| Status | Flag | Description |
| --- | --- | --- |
| Beta | `--enable-composition-revisions` |Enable support for CompositionRevisions |
| Alpha | `--enable-composition-functions` | Enable support for Composition Functions. |
| Alpha | `--enable-composition-webhook-schema-validation` | Enable Composition validation using schemas.
| Alpha | `--enable-environment-configs` | Enable support for EnvironmentConfigs. |
| Alpha | `--enable-external-secret-stores` | Enable support for External Secret Stores. |
{{< /table >}}
{{< /expand >}}

> These can be set either in the `values.yaml` file or at install time using the

Reviewer: ```suggestion
Set these flags either in the `values.yaml` file or at install time using the
```
